packages/resources/discourse-openapi-spec-edited.json

```diff
@@ -0,0 +1,1068 @@
+{
```

Comment:
Please just provide a link to the following information in the either the main readme or one specific to Discourse with some key points, we don't need the to have the docs live within our repo

Reply:
Hm, this isn't the docs.. This is an [Open API](https://swagger.io/resources/open-api/) spec (taken from the official Discourse site and prepared a little) which I used to auto-generate the React components with restful-react. I am happy to remove it, but IMO that won't help when someone would need to run the code-gen tool again.

Reply:
Not familiar enough with restful-react to have an informed opinion, but my initial instinct says its too complicated for our uses, might be better just to make our own git calls and make our own react components instead of having it be generated for us. I'll dive deeper into this later though.